README.md

```diff
@@ -182,7 +182,7 @@ $ ./out/cmake/uv_run_tests_a  # static library build
 To build with GYP, first run:
 
 ```bash
-$ git clone https://chromium.googlesource.com/external/gyp build/gyp
+$ git clone https://github.com/libuv/gyp build/gyp
```

Comment:
Chromium's copy of gyp contains patches for z/OS (e.g. https://chromium-review.googlesource.com/c/external/gyp/+/644712/) which are not in https://github.com/nodejs/node/tree/d0413ae8f92aa0591506ff115f21ec0ae2591e62/tools/gyp that was used to seed https://github.com/libuv. This isn't evident from CI as currently the a copy of gyp served from Jenkins (https://github.com/nodejs/build/issues/2172) is used.